app/views/layouts/_header.html.haml

```diff
@@ -12,8 +12,10 @@
   %nav.collapse.navbar-collapse
     %ul.nav.navbar-nav.navbar-right.navbar-iconic
       %li{:class => "drawer-pf-trigger dropdown"}
-        %a{:class => "nav-item-iconic drawer-pf-trigger-icon", "ng-click" => "vm.toggleNotificationsList()"}
-          %span{:title => _("Notifications"), "ng-class" => "{'fa fa-bell': vm.newNotifications, 'fa fa-bell-o': !vm.newNotifications}"}
+        %a{:class => "nav-item-iconic drawer-pf-trigger-icon", :title => "{{vm.notificationsIndicatorTooltip}}", "ng-click" => "vm.toggleNotificationsList()"}
+          %span{"ng-class" => "{'fa fa-bell': vm.newNotifications, 'fa fa-bell-o': !vm.newNotifications}"}
```

Comment:
Is using multiple classes officially supported here? The angular docs mention only the key is used as class name.

So maybe better written as `%span.fa{"ng-class" => "{'fa-bell': vm.newNotifications, 'fa-bell-o': !vm.newNotifications}"}` ?